### ISSUE TITLE
FortiGate configuration

### ISSUE THREAD

**Issue by andbus** [pfitchie/graylog3.Fortigate6xContentPack#2]
Can I ask for the README entry to be extended with FortiGate configuration information? What parameters set on FortiGate work with this package?

**Comment by raatti** [comment]
Indeed, it seems that formatting needs to specified also from fortigate side..
